Automattic/newspack-popups
# Feat/polyfill amp scripts

### All Submissions:






### Changes proposed in this Pull Request:



Partially addresses https://github.com/Automattic/newspack-popups/issues/193, polyfiling the use of `amp-animation`, `amp-position-observer`, `amp-bind`.

Note that this PR contains a refactor of file structure, so the actual diff size is smaller. There were virtually no changes to code in `src/view/analytics.js` and `src/view/form.js`.

### How to test the changes in this Pull Request:

1. Turn off the AMP Plugin
2. Test a scroll-triggered prompt (should display, with an animation, on scroll)
3. Test an overlay prompt (should animate into view)
4. Both should work as before

### Other information:
